### ISSUE TITLE
xdrip+/g5 transmitter reset/ t slim 2x

### ISSUE THREAD

**Issue by 4ElizabethS** [StephenBlackWasAlreadyTaken/xDrip#246]
Is it possible to reset g5 transmitter & have data go to t slim 2x pump?
How?
ty!!!

**Comment by schwartz914** [comment]
Yes, 3rd post down in this link: https://forum.fudiabetes.org/t/resetting-g5-transmitter-with-xdrip/6209.

Just to note. I did this about 47 days ago. I use the X2 and G5
